src/Hpack/Render.hs

```diff
@@ -186,7 +186,7 @@ renderBenchmark (name, sect) =
     (renderExecutableSection [Field "type" "exitcode-stdio-1.0"] sect)
 
 renderExecutableSection :: [Element] -> Section Executable -> [Element]
-renderExecutableSection extraFields = renderSection renderExecutableFields extraFields [defaultLanguage]
+renderExecutableSection extraFields = renderSection renderExecutableFields extraFields []
```

Comment:
I think it's now possible to simplify `renderSection` (remove the last argument). Or isn't it?

Reply:
It is possible to simplify, and I will do that.